templates/item/preview.macros.j2

```diff
@@ -31,24 +32,35 @@
   Display color palettes based on JSON map output.
 #}
 {% macro color_palette(data) %}
-<div data-sassdoc="color-palette">
-  {% for name, value in data %}
-    <div data-sassdoc-color="{{ name }}">
-      <div data-sassdoc="color-swatch">
-        <div data-sassdoc="transparency-grid"></div>
-        <div data-sassdoc="color-overlay" style="background-color: {{ value.output }};"></div>
-      </div>
-      <div data-sassdoc="color-info">
-        <strong data-sassdoc="color-name">{{ name }}</strong>
-        {% for type, color in value %}
-          <div data-sassdoc="color-value">
-            {{ color }}
+  {% call show.item_section('color-palette', 'Color Preview') %}
+    <div class="color-palette">
+      {% for value, items in data|list|groupby('value') %}
+        <div class="color-preview">
+          <div class="color-swatch">
+            <div class="transparency-grid"></div>
+            <div class="color-overlay" style="background-color: {{ value }};"></div>
           </div>
-        {% endfor %}
-      </div>
+          <div class="color-info">
+            <strong class="color-name">
+              {% for item in items %}
+                <span class="color-name-option">{{ item.key }}{% if not loop.last %},&nbsp;{% endif %}</span>
+              {% endfor %}
+            </strong>
```

Comment:
I still think it would be cool to have the css syntax here as a copyable line too: 
```
color('{{ item.key }}');
```

Reply:
Yeah, that's a cool idea – and I can imagine how it would work. But it sounds like a new feature to me, with a setting of it's own. Want to file an issue for that?